bin/system-tests/tests/test-arm-exes.bash

```diff
@@ -314,6 +314,8 @@ test_arm_bounds_check () {
 
     print_header "Checking ${TEST_DIR}"
 
+    run_make "make clean -C ${TEST_DIR}" 0
+
```

Comment:
Why is this needed for this test and not the others?

Reply:
From what I can tell it was actually being run in most or all of the other tests, but not this one. I noticed that it was returning immediately when I would run the system tests, and realized it was running the patched binary that was left from a previous run.